src/config.c

```diff
@@ -535,6 +552,8 @@ void loadServerConfigFromString(char *config) {
             }
         } else if (!strcasecmp(argv[0],"loadmodule") && argc >= 2) {
             queueLoadModule(argv[1],&argv[2],argc-2);
+        } else if (strchr(argv[0], '.')) {
+            server.module_configs_queue = listAddNodeTail(server.module_configs_queue, sdsnew(lines[i]));
```

Comment:
If we don't do that, how do we guarantee there are no name conflicts?

Reply:
The code doesn't require the '.', it was just a mechanism to help prevent conflicts. It does prevent us from adding configs with .s in the future, but I'm not sure why we would ever do that.

Reply:
looking at the code i don't see why it prevents us to add such configs in the future. we go look for module configs only after we're done trying to match the built in ones.
i think this can remain as is.
i.e. module configs have to have `.`, but it doesn't mean that they're the only ones.
i don't see any problem with conflicts this way.